addons/binding/org.openhab.binding.vallox/src/main/java/org/openhab/binding/vallox/internal/serial/ValloxProperty.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.vallox.internal.serial;
+
+/**
+ * Enum that represent channels which represent
+ * properties of the vallox unit. Enum used here because
+ * there are quite many different literals and as Enum it
+ * can be used in switch-statements.
+ *
+ * @author Hauke Fuhrmann - Initial contribution
+ *
+ */
+public enum ValloxProperty {
+    // sensor data
+    FanSpeed, // VALLOX_VARIABLE_FAN_SPEED
```

Comment:
This is one I could not address so far: the binding has quite a number of channels as you might have noticed. They are directly corresponding to this Enum-Literals. In order to not have many many long lists of channel IDs, Enum Literals, Telegram constants and so on, I try to reuse the enum to also denote the channel ID. So I can call ValloxProperty.FanSpeed.name() which will be a valid channel ID also.
I know this is not following naming conventions for enums, but I have not the time to correct this right now just for formal reasons.
I know that the mapping of channels to these Properties to telegrams I receive and send to the physical device is not perfectly done. However, refactoring this would make sense to completely remove that Enum and add some subclassing to telegrams with special telegram classes for each channel and/or telegram handling everything including the channel. However, this is not something I can do at the moment. Hope we can get this initial version despite this issue...